packages/react-native-sdk/src/providers/MediaStreamManagement.tsx

```diff
@@ -94,36 +95,35 @@ export const MediaStreamManagement = ({
         call?.camera.disable();
       }
     }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [initialAudioEnabled, initialVideoEnabled]);
+  }, [call, initialAudioEnabled, initialVideoEnabled]);
 
   const toggleInitialAudioMuteState = useCallback(() => {
     if (
       !isMicPermissionGranted$.getValue() &&
       call?.microphone.state.status === 'disabled'
     ) {
-      Alert.alert('Microphone permission not granted, can not enable audio');
+      Alert.alert(t('Microphone Permission Required To Enable Audio'));
       return false;
     }
 
     call?.microphone.state.status === 'disabled'
       ? call?.microphone.enable()
       : call?.microphone.disable();
-  }, [call]);
+  }, [call, t]);
 
   const toggleInitialVideoMuteState = useCallback(() => {
     if (
       !isCameraPermissionGranted$.getValue() &&
       call?.camera.state.status === 'disabled'
     ) {
-      Alert.alert('Camera permission not granted, can not enable video');
+      Alert.alert(t('Camera Permission Required To Enable Video'));
       return false;
     }
 
     call?.camera.state.status === 'disabled'
       ? call?.camera.enable()
       : call?.camera.disable();
-  }, [call]);
+  }, [call, t]);
```

Comment:
Not sure if we should pass the `t` function in the usecallback dep array. Other than it the PR looks good.